corehq/apps/users/management/commands/copy_web_permissions_to_all_apps.py

```diff
@@ -0,0 +1,29 @@
+from django.core.management.base import BaseCommand
+
+from corehq.apps.users.models import UserRole
+from corehq.util.couch import DocUpdate, iter_update
+from corehq.util.log import with_progress_bar
+
+
+def _copy_permissions(role_doc):
+    role = UserRole.wrap(role_doc)
+    permissions = role_doc['permissions']
+    if permissions.get('access_all_apps') != permissions.get('view_web_apps'):
+        role.permissions.access_all_apps = permissions.get('view_web_apps', True)
+    if permissions.get('allowed_app_list') != permissions.get('view_web_apps_list'):
+        role.permissions.allowed_app_list = permissions.get('view_web_apps_list', [])
+    return DocUpdate(role)
```

Comment:
One behavioral quirk I just thought of - any values not in the database will be set to the default by `wrap`.  For example:
```python
>>> role = UserRole.wrap({})
>>> role.permissions.view_apps
False
```
This will register as a change to the document, and `iter_update` will save it.  I think that's desirable, though.

Reply:
Yeah. The schema should match with the actual data.